fastapi_websocket_rpc/websocket_rpc_client.py

```diff
@@ -93,13 +106,17 @@ async def recv(self):
     async def close(self, code: int = 1000):
         if self._websocket is not None:
             # Case opened, we have something to close.
-            self._websocket.close(code)
+            await asyncio.get_event_loop().run_in_executor(
+                None, self._websocket.close, code
+            )
```

Comment:
why does it matter if this code is blocking the event loop, it closes the websocket, why would you want anything else running in the same time ?

Reply:
I'm keeping with @antoinetran changes here - see other methods on this handler